packages/radix-vue/src/Calendar/CalendarRoot.vue

```diff
@@ -169,6 +173,8 @@ const {
   isDateUnavailable: propsIsDateUnavailable,
   calendarLabel,
   defaultValue,
+  nextPage: propsNextPage,
+  prevPage: propsPrevPage,
```

Comment:
why declare the name with `propsXXX` if there's no naming conflict? 🤔

Reply:
ESLint said I had a naming conflict with this line: https://github.com/radix-vue/radix-vue/blob/6ca465cc92140c142404f42559d41c1a0c6284b2/packages/radix-vue/src/Calendar/CalendarRoot.vue#L213 (cannot redeclare block-scoped variable 'nextPage')